contents/newsletter/good-taste-great-products.md

```diff
@@ -0,0 +1,209 @@
+
+Radio buttons and checkboxes are another timeless example.
+
+![](https://res.cloudinary.com/dmukukwp6/image/upload/5905eff1_ed7f_44be_8b17_5e14098fdcf7_1546x874_c10bdfea0d.jpg)
+When we see the circular bullets of the **radio button** , we know that we can choose _only one_ of the presented options.
+
+Whereas the **checkbox** tells us _multiple_ selections are valid. 
+
+![](https://res.cloudinary.com/dmukukwp6/image/upload/51612b56_30a6_4f00_97d8_a2089a1ad2e8_1284x910_b51b572a72.jpg)
+
+We understand these affordances at a glance. 
+
+We can validate our assumptions by playing with the controls, but their shapes telegraph their function before we interact, saving us time.
+
+Here are a few more you still see today:
+
+  *  **Buttons** – clickable regions – have outlines and often distinct colors
+
+  *  **Text fields** have borders and sometimes inner shadows, along with placeholder text to provide examples of what you should enter
+
+  * A **pop-up menu** often sports a small triangle, pointed downward, to suggest that clicking it will reveal additional options
+
+You need to play with lots of software to stay abreast of affordances – it’s vital for learning the design vocabulary users will expect and recognize.
+
+Make a list of all the affordances you observe in software you use frequently. Do the same for products that are similar to what you’re building and look for patterns. 
+
+The more you do this, the more you’ll instinctively build them into your work, thus making your software feel effortless and familiar. 
+
+> **Recommended reading:** [_Design is Storytelling_](https://www.amazon.com/Design-Storytelling-Ellen-Lupton/dp/194230319X) by Ellen Upton is a fun treatise on the power of narrative to help software communicate with users.
+
+* * *
+
+##  **2. Detect and eliminate obvious jank**
+
+You don’t need to be the most talented designer on earth to have taste.
+
+Instead, you need a reliable sense of what is janky and what is nice, and the ability to nudge a product in the nicer direction.
+
+People who encounter janky software treat it with suspicion, and eventually disdain. They lose trust in the people building it, and won’t recommend it.
+
+In contrast, pleasant software suggests care and attention, and earns loyalty. Beneath the surface, it is assumed, yet more care is lurking. Decently designed products feel more trustworthy.
+
+You can spend a lifetime learning design, but you can learn how to spot jankiness right now.
+
+###  **Make friends with the grid**
+
+Grids make things look tidy and deliberate. If something feels _vaguely janky,_ it might be because visual elements aren’t lining up according to any rules. 
+
+![](https://res.cloudinary.com/dmukukwp6/image/upload/3a940cd7_5227_472c_bf55_d47193d8010d_2020x962_40f183c715.png)
```

Comment:
In dark mode, you can't see the text on this image sadly. Maybe needs a separate version